coreplugins/cloudimport/api_views.py

```diff
@@ -100,11 +101,6 @@ def get(self, request):
 
 
 def import_files(task_id, files):
-    import requests
```

Comment:
This is exactly why I dislike linters. You're breaking this function because it's executed in an async context, where imports are **not** available from the top module level. But the linter can't possibly know this and suggests to move the imports at the top.